huaweicloud/services/rms/resource_huaweicloud_rms_policy_assignment.go

```diff
@@ -0,0 +1,578 @@
+package rms
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"reflect"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/rms/v1/policyassignments"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+const (
+	AssignmentTypeBuiltin = "builtin"
+	AssignmentTypeCustom  = "custom"
+
+	AssignmentStatusDisabled   = "Disabled"
+	AssignmentStatusEnabled    = "Enabled"
+	AssignmentStatusEvaluating = "Evaluating"
+)
+
+func ResourcePolicyAssignment() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceePolicyAssignmentCreate,
+		ReadContext:   resourceePolicyAssignmentRead,
+		UpdateContext: resourceePolicyAssignmentUpdate,
+		DeleteContext: resourceePolicyAssignmentDelete,
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(5 * time.Minute),
+			Update: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The region where the policy assignment is located.",
+			},
+			"domain_id": {
```

Comment:
the domain_id is a constant value for users, and we cat get it from config, so I think we should not define it in schema

Reply:
sure, this parameter has been removed.